computation-slurm/src/main/java/com/powsybl/computation/slurm/SlurmComputationManager.java

```diff
@@ -359,13 +359,17 @@ private SlurmExecutionReport generateReport(Map<Long, Command> jobIdCommandMap,
                 m.find();
                 long jobId = Long.parseLong(m.group());
                 m.find();
-                int executionIdx = Integer.parseInt(m.group());
-                m.find();
                 int exitCode = Integer.parseInt(m.group());
+                long mid = jobId;
+                int executionIdx = 0;
+                if (!jobIdCommandMap.containsKey(jobId)) {
```

Comment:
In the future, this kind of logic should be encapsulated in slurm task or even in "task array" implementations (via job array or master/"slaves").